package.json

```diff
@@ -20,9 +20,9 @@
     "dotenv": "16.0.3",
     "mocha-junit-reporter": "^2.2.0",
     "sauce-testrunner-utils": "1.0.0",
-    "testcafe": "2.3.1",
+    "testcafe": "^2.3.1",
```

Comment:
We probably should keep that fixated to a specific version 🤔

Reply:
Yup, that was un unexpected change.